bin/src/command/orders.rs

```diff
@@ -806,45 +775,41 @@ impl CommandServer {
 
         return_processing(
             self.command_tx.clone(),
-            request_identifier.clone(),
+            client_id.clone(),
             "Reloading configuration, sending config messages to workers...",
         )
         .await;
 
-        for message in new_config.generate_config_messages() {
-            if let CommandRequestOrder::Proxy(order) = message.order {
-                if self.state.dispatch(&order).is_ok() {
-                    diff_counter += 1;
+        for order in new_config.generate_config_messages() {
+            if self.state.dispatch(&order.content).is_ok() {
+                diff_counter += 1;
 
-                    let mut found = false;
-                    let id = format!(
-                        "LOAD-STATE-{}-{}",
-                        &request_identifier.request, diff_counter
-                    );
+                let mut found = false;
+                let id = format!("LOAD-STATE-{}-{}", &order.id, diff_counter);
 
-                    for ref mut worker in self.workers.iter_mut().filter(|worker| {
-                        worker.run_state != RunState::Stopping
-                            && worker.run_state != RunState::Stopped
-                    }) {
-                        let worker_message_id = format!("{}-{}", id, worker.id);
-                        worker.send(worker_message_id.clone(), *order.clone()).await;
-                        self.in_flight
-                            .insert(worker_message_id, (load_state_tx.clone(), 1));
+                for ref mut worker in self.workers.iter_mut().filter(|worker| {
+                    worker.run_state != RunState::Stopping && worker.run_state != RunState::Stopped
+                }) {
+                    let worker_message_id = format!("{}-{}", id, worker.id);
+                    worker
+                        .send(worker_message_id.clone(), order.content.clone())
+                        .await;
+                    self.in_flight
+                        .insert(worker_message_id, (load_state_tx.clone(), 1));
 
-                        found = true;
-                    }
+                    found = true;
+                }
 
-                    if !found {
-                        // FIXME: should send back error here
-                        error!("no worker found");
-                    }
+                if !found {
```

Comment:
To me, we could safely remove this. As this case could not produce except in development environment and I prefer to add a log that indicate that we send a message to a worker instead in trace or debug level.

Reply:
To me, sending an error is the best thing to do. It will be formatted, trickled up to the client and duly logged.